src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -210,11 +210,11 @@ ov::runtime::Tensor create_and_fill_tensor_consistently(const ov::element::Type
 constexpr double eps = std::numeric_limits<double>::epsilon();
 
 inline double less(double a, double b) {
-    return (b - a) > (std::fmax(std::fabs(a), std::fabs(b)) * eps);
+    return a < b;
```

Comment:
I think, we should use eps here because float difference. 
`std::fabs(a - b) > eps && a < b`

Reply:
done